c_src/unifex/cnode/unifex/unifex.h

```diff
@@ -44,12 +45,35 @@ static inline void *unifex_alloc(size_t size) { return malloc(size); }
 
 static inline void unifex_free(void *pointer) { free(pointer); }
 
+/**
+ * Env
+ * 
+ * These functions are introduced to achieve interoperability between NIF and C node.
+ * They does nothing.
```

Comment:
They don't really do nothing, as unifex_alloc_env returns env ;) I'm not sure if we should document that behaviour as it may change

Reply:
Done, but I think we should consider documenting somehow C API. In case of NIF we generate code with documentation but I am not sure if this is the best solution as to generate such a code someone need to write some other code first. I think we could move all the documentation into one place.